public/js/practise.js

```diff
@@ -4,16 +4,16 @@ let number = DoubleNumber(12);
 function DoubleNumber(number1) {
     let result = number1 + number1;
 
-    return result;
+    return "the result is " + (number1 + number1);
```

Comment:
You can remove from **line 7** this part _(number1 + number2)_ and place the  **result** variable.